src/LibraryManager.Vsix/Commands/CommandTable/VSCommandTable.en.vsct

```diff
@@ -19,7 +19,7 @@
         <CommandFlag>DefaultInvisible</CommandFlag>
         <CommandFlag>DynamicVisibility</CommandFlag>
         <Strings>
-          <ButtonText>Client-Side Library...</ButtonText>
+          <ButtonText>Client-Side &amp;Library...</ButtonText>
```

Comment:
This item only currently appears in a submenu called "Add" (with accelerator key "d").
Should not appear on the same submenu as "Restore Client-Side Libraries".

Reply:
All letters of "Restore" are already being used by:
- (R) Properties
- (E) Enable Restore Client-Side Libraries on Build (my new feature)
- (S) Scope to this
- (T) Cut
- (O) Open

"C" is also being used by "Clean Client-Side Libraries".

So "L" seems the best choice.